content/docs/user-guide/data-management/remote-storage/azure-blob-storage.md

```diff
@@ -219,3 +219,14 @@ See `dvc remote modify` for more command usage details.
 
 [Azure documentation]:
   https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-blob#other-client--per-operation-configuration
+
+### Azure Remote Configuration 
+
+- `"exclude_environment_credential"`: Bool. If `true`, excludes the environment credential source for Azure Remote.
```

Comment:
please keep the style exactly the same as with the list above: 

`:` -> `-`

start the description from the low case. no need to specify type.

Reply:
removed if statements and kept `bool` , is that fine?